geozero-shp/src/property_processor.rs

```diff
@@ -7,29 +7,29 @@ impl FeatureProperties for ShapeRecord {
     /// Process feature properties.
     fn process_properties<P: PropertyProcessor>(&self, processor: &mut P) -> Result<bool> {
         let mut finish = false;
-        for (i, (name, value)) in self.record.iter().enumerate() {
+        for (i, (name, value)) in self.record.clone().into_iter().enumerate() {
```

Comment:
Instead of `clone`, `as_ref` can be used.

Reply:
Ok thanks for the tip. I just pushed the changes.

This is my second PR so thanks for bearing with me, should I do anything else?